Containers/Ubuntu-22/Dockerfile

```diff
@@ -168,7 +168,7 @@ RUN update-alternatives \
 FROM build AS test
 
 ARG QEMU_URL="https://gitlab.com/qemu-project/qemu.git"
-ARG QEMU_BRANCH="v8.0.0"
+ARG QEMU_BRANCH="v9.0.0"

```

Comment:
Not necessary now, but in the future, we might want to consider adding this to `Version.njk`.